herd/RISCVSem.ml

```diff
@@ -255,10 +257,20 @@ module
       let build_semantics _ ii =
         M.addT (A.next_po_index ii.A.program_order_index)
           begin match ii.A.inst with
+          | RISCV.INop-> B.next1T ()
+          | RISCV.Ret -> M.unitT () >>! B.Exit
+          | RISCV.OpI (RISCV.LUI,r1,_,k) ->
```

Comment:
This encoding of the `LUI` instruction is surprising. Moreover the instruction `lui r,k` is printed as `lui r,x0,k`.